### ISSUE TITLE
We need a Scene server that holds all information about scenes

### ISSUE THREAD

**Issue by BrianAmadori** [decentraland/explorer#2006]
![image](https://user-images.githubusercontent.com/6875814/111693202-17cb0480-880f-11eb-9aff-e3d51e616ce9.png)

**Comment by olavra** [comment]
@pravusjif are you working on this?

**Comment by menduz** [comment]
Can you provide more details? Probably we have it in the catalyst, but it is hard to know based on the title only

**Comment by pravusjif** [comment]
> @pravusjif are you working on this?

No, it's the first time I read about this, kernel normaly fetches the scenes data hitting some lambdas in the catalysts i think, maybe a more centralized way could be faster and more optimized.

**Comment by menduz** [comment]
catalysts are fast enough for our use cases, they act as central servers, we just have many of them available. please describe the need and what needs to be improved, that would yield better results

**Comment by BrianAmadori** [comment]
I think this is related to where we are going to store the unpublished scenes (projects) that were going to appear in the builder in-world panel. Going to update the description to add more details.
